pages/u.tsx

```diff
@@ -74,6 +75,9 @@ export const getServerSideProps: GetServerSideProps<ProfileProps> = async ({ req
 		updateUser(session.user.id!, { aliases: aliases }).catch(err => {
 			console.error(`[Profile] Could not update aliases for user (userId: ${session.user.id})`, err)
 		})
+		updateAliasesTableOnUserLogin(aliases, session.user.id!).catch(err => {
```

Comment:
the name of the function is incorrect. This function doesn't only work when the user logs in. It can work whenever we want, as long as aliases are provided and session is available. 

I recommend a simple name `updateAliasesForUser`.